src/funjson.c

```diff
@@ -64,6 +65,36 @@ json_escape_string(char *input)
   return buff;
 }
 
+/** Unescape a string for use as a JSON string. Returns a calloced buffer under 'json.string.partialbuf'. **/
+char * json_unescape_latin1string(char *latin1, int len)
+{
+  char *c;
+  char *partialbuf = mush_calloc(len*6,sizeof(char),"json.string.partialbuf");
+  char *bpp = partialbuf;
+  int pblen = 0;
```

Comment:
Doesn't look like this actually gets used?

Reply:
Aye, it used to in a preview version.